src/document_handler.rb

```diff
@@ -0,0 +1,30 @@
+require 'telegramAPI'
+
+class DocumentHandler
+  attr_reader :document, :api
+
+  def initialize(request_handler)
+    @token = ENV['TOKEN']
+    @document = request_handler.message.document
+
+    @api = TelegramAPI.new(@token)
+  end
+
+  def translate_document
+    File.new('translated.txt', 'w').puts translate_content
+  end
+
+  private
+
+  def get_file_content
+    open("https://api.telegram.org/file/bot#{@token}/#{api.getFile(document.file_id)['file_path']}") do |f|
```

Comment:
It's a good idea to use `URI` to assemble an url, plus, it would be cool to move that url to initializer.